learn-more.html

```diff
@@ -1,13 +1,246 @@
+          veritatis voluptate beatae nobis nulla incidunt aliquid provident ipsum fugit cusantium, quibusdam sapiente
+          culpa modifacilis quidem necessitatibus impedit quod.</p>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias laudantium veritatis voluptate beatae
+          nobis nulla incidunt aliquid provident ipsum fugit cusantium, quibusdam sapiente culpa modifacilis quidem
+          necessitatibus impedit quod.Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias laudantium
+          veritatis voluptate beatae nobis nulla incidunt aliquid provident ipsum fugit cusantium, quibusdam sapiente
+          culpa modifacilis quidem necessitatibus impedit quod.</p>
+      </div>
+
+    </column>
+
+  </section>
+
+
+  <section id="footer">
+    <div id="footer-content">
+      <div class="footer-title">
+        <h3 class="section-title">QUESTIONS OR COMMENTS?&nbsp; <b> GET IN TOUCH</b></h3>
+      </div>
+      <div class="footer-lower">
+        <div id="footer-form-container">
+          <form>
+            <div id="name-email">
+              <input class="name-email-input" type="text" placeholder="NAME" />
+              <input class="name-email-input" type="text" placeholder="EMAIL">
+            </div>
+            <textarea class="message-input"></textarea>
+            <input class="orange-btn" type="submit">
+          </form>
+        </div>
+        <div id="footer-contact-info">
+          <p>Lorem ipsum dolor, sit amet consectetur isicing elit. Quaerat aliquam veniam nam cum. Magnnesciunt facilis
+            eligendi libero sunt, iste a, corrupti quas ellendus optio cupiditate consectetur mollitia quae ab?
+          </p>
+          <div id="icon-column">
+            <div id="footer-column-2">
+              <div id="contact-links">
+
+                <div id="left-contacts">
+                  <div class="contacts">
+                    <img class="contact-img-wrapper" src="footerlink-home.png">
+                    <p>Lorem, ipsum dolor sit amet consectetur voluptatum nemo </p>
+                  </div>
+                  <div class="contacts">
+                    <img class="contact-img-wrapper" src="footerlink-phone.png">
+                    <p>800-555-1212</p>
+                  </div>
+                  <div class="contacts">
+                    <img class="contact-img-wrapper" src="footerlink-email.png">
+                    <p>Lorem, ipsum dolor sit amet consectetur voluptatum nemo </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div id="footer-column-3">
+              <div id="right-contacts">
+                <div class="contacts">
+                  <img class="contact-img-wrapper" src="footerlink-twitter.png">
+                  <p>Lorem, ipsum dolor sit amet consectetur voluptatum nemo </p>

```

Comment:
Your icons and images are broken!  Presumably from a restructuring of your directory;  Your `img` sources are in the **img** folder now, but these aren't pointing there.  Same for all images on this page.  Easy fix, but definitely one worth making.

Reply:
The anchor tags are dead too! Fix those up and you'll be right as rain.

Reply:
Fixed! You were right, I just missed this page with renaming the icons once I created the folder for images. I did change the anchor tags, but they are the ones that link to the page that they are on.